abci/checktx/mempool_parity_check_tx.go

```diff
@@ -98,10 +106,99 @@ func (m MempoolParityCheckTx) CheckTx() CheckTx {
 			}
 		}
 
+		sdkCtx := m.GetContextForTx(req)
```

Comment:
Why do we need to do this?

Reply:
we need a context to get consensus params

Reply:
and to call `Match` on the lanes

Reply:
I see we're wrapping the `CheckTx` handler and don't have access to the base-app state directly..